src/world/mob/player/action/input-command-action.ts

```diff
@@ -35,14 +35,15 @@ const commands: { [key: string]: commandHandler } = {
 
```

Comment:
the `::move` command was previously crashing due to the new object chunk packets. This fixes it.